main/translate.c

```diff
@@ -1056,7 +1058,15 @@ static char *handle_show_translation_table(struct ast_cli_args *a)
 
 			if (x >= 0 && y >= 0 && matrix_get(x, y)->step) {
 				/* Actual codec output */
-				ast_str_append(&out, 0, "%*u", curlen + 1, (matrix_get(x, y)->table_cost/100));
+				if (a->argv[3] && !strcasecmp(a->argv[3], "comp")) {
+					compCost = matrix_get(x, y)->comp_cost;
+					if(compCost == 0 || compCost == 999999)
```

Comment:
if (compCost

and as well put brackets { } for if and else

Reply:
@jcolp both done, see updated commit